level-ttl.js

```diff
@@ -183,7 +208,7 @@ function batch (db, arr, options, callback) {
     on  = []
     off = []
     arr.forEach(function (entry) {
-      if (!entry || entry.key === null || entry.key === undefined)
+      if (!entry || entry.key === null || entry.key === undefined || entry.key.indexOf('\xffttl\xff') > -1)
```

Comment:
level-ttl does several batch saves using its own overloaded function. For example ...
```
[ { type: 'put', key: 'ÿttlÿfoo', value: '1421846940121' },
  { type: 'put', key: 'ÿttlÿ1421846940121!foo', value: 'foo' } ]
```

This prevents a loop from occuring when those keys are added.

I think this happens because `db._ttl.sub` is being used and when sub is intialized its passed the `db` argument and don't know about the original batch and put functions and uses the overloaded ones.

Put is never called with \xffttl\xff prefixed keys so I didn't add it there, but it might not hurt to prevent loops from happening.